src/valkey-cli.c

```diff
@@ -4993,7 +4991,7 @@ clusterManagerMoveSlot(clusterManagerNode *source, clusterManagerNode *target, i
          * unblocked with the role change error. */
         success = clusterManagerSetSlot(source, target, slot, "node", err);
         if (!success && err) {
-            const char *acceptable[] = {"ERR Please use SETSLOT only with masters.", "UNBLOCKED"};
+            const char *acceptable[] = {"ERR Please use SETSLOT only with ", "UNBLOCKED"};
```

Comment:
missing this?
```suggestion
            const char *acceptable[] = {"ERR Please use SETSLOT only with primaries.", "UNBLOCKED"};
```

Reply:
Actually I can't change the error string as that might break some clients. I will add both strings (primaries and masters) next.